### ISSUE TITLE
ResourceConflictException when 'make deploy' target is ran

### ISSUE THREAD

**Issue by achave11** [DataBiosphere/azul#1547]
```$ make deploy
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C lambdas
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C indexer deploy
python /Users/abrahamchavez/PycharmProjects/azul/scripts/check_branch.py
python .chalice/config.json.template.py .chalice/config.json
Creating .chalice/config.json
python lambda-policy.json.template.py lambda-policy.json
Creating lambda-policy.json
python /Users/abrahamchavez/PycharmProjects/azul/scripts/manage_chalice_deployed.py indexer download
2020-02-06 07:45:20,516 INFO    MainThread: Found credentials in environment variables.
2020-02-06 07:45:20,618 INFO    MainThread: Downloading s3://edu-ucsc-gi-singlecell-azul-dev-config/azul-indexer-achave11/deployed.json to .chalice/deployed/achave11.json
rm -rf vendor/jsonobject*
unzip /Users/abrahamchavez/PycharmProjects/azul/lambdas/.wheels/jsonobject-0.9.9-cp36-cp36m-linux_x86_64.whl -d vendor
Archive:  /Users/abrahamchavez/PycharmProjects/azul/lambdas/.wheels/jsonobject-0.9.9-cp36-cp36m-linux_x86_64.whl
  inflating: vendor/jsonobject/__init__.py  
  inflating: vendor/jsonobject/api.cpython-36m-x86_64-linux-gnu.so  
  inflating: vendor/jsonobject/base.cpython-36m-x86_64-linux-gnu.so  
  inflating: vendor/jsonobject/base_properties.cpython-36m-x86_64-linux-gnu.so  
  inflating: vendor/jsonobject/containers.cpython-36m-x86_64-linux-gnu.so  
  inflating: vendor/jsonobject/exceptions.py  
  inflating: vendor/jsonobject/properties.cpython-36m-x86_64-linux-gnu.so  
  inflating: vendor/jsonobject/utils.cpython-36m-x86_64-linux-gnu.so  
  inflating: vendor/jsonobject-0.9.9.dist-info/LICENSE  
  inflating: vendor/jsonobject-0.9.9.dist-info/METADATA  
  inflating: vendor/jsonobject-0.9.9.dist-info/WHEEL  
  inflating: vendor/jsonobject-0.9.9.dist-info/top_level.txt  
  inflating: vendor/jsonobject-0.9.9.dist-info/RECORD  
python -m azul.changelog vendor
python /Users/abrahamchavez/PycharmProjects/azul/scripts/manage_lambda_iam_role.py create indexer lambda-policy.json
2020-02-06 07:45:21,610 INFO    MainThread: Found credentials in environment variables.
2020-02-06 07:45:23,311 INFO    MainThread: Creating IAM Role azul-indexer-achave11.
2020-02-06 07:45:23,311 INFO    MainThread: Role azul-indexer-achave11 already up-to-date.
chalice deploy --stage achave11
Creating deployment package.

Could not install dependencies:
jsonobject==0.9.9
You will have to build these yourself and vendor them in
the chalice vendor folder.

Your deployment will continue but may not work correctly
if missing dependencies are not present. For more information:
http://chalice.readthedocs.io/en/latest/topics/packaging.html

Updating lambda function: azul-indexer-achave11-indexercachehealth
Updating lambda function: azul-indexer-achave11-index
Updating lambda function concurrency limit: azul-indexer-achave11-index
Subscribing azul-indexer-achave11-index to SQS queue azul-notify-achave11
Traceback (most recent call last):
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/chalice/deploy/deployer.py", line 342, in deploy
    return self._deploy(config, chalice_stage_name)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/chalice/deploy/deployer.py", line 355, in _deploy
    self._executor.execute(plan)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/chalice/deploy/executor.py", line 31, in execute
    self._default_handler)(instruction)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/chalice/deploy/executor.py", line 43, in _do_apicall
    result = method(**final_kwargs)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/chalice/awsclient.py", line 962, in create_sqs_event_source
    max_attempts=self.LAMBDA_CREATE_ATTEMPTS
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/chalice/awsclient.py", line 1148, in _call_client_method_with_retries
    response = method(**kwargs)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/botocore/client.py", line 661, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.ResourceConflictException: An error occurred (ResourceConflictException) when calling the CreateEventSourceMapping operation: An event source mapping with SQS arn (" arn:aws:sqs:us-west-2:122796619775:azul-notify-achave11 ") and function (" azul-indexer-achave11-index ") already exists. Please update or delete the existing mapping with UUID 2dba5c66-5f29-4e2a-91e3-777f8b54802d

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/chalice/cli/__init__.py", line 512, in main
    return cli(obj={})
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/chalice/cli/__init__.py", line 205, in deploy
    deployed_values = d.deploy(config, chalice_stage_name=stage)
  File "/Users/abrahamchavez/PycharmProjects/azul/.venv/lib/python3.6/site-packages/chalice/deploy/deployer.py", line 344, in deploy
    raise ChaliceDeploymentError(e)
chalice.deploy.deployer.ChaliceDeploymentError: ERROR - While deploying your chalice application, received the following error:

 An error occurred (ResourceConflictException) when calling the 
 CreateEventSourceMapping operation: An event source mapping with SQS arn (" 
 arn:aws:sqs:us-west-2:122796619775:azul-notify-achave11 ") and function (" 
 azul-indexer-achave11-index ") already exists. Please update or delete the 
 existing mapping with UUID 2dba5c66-5f29-4e2a-91e3-777f8b54802d



make[2]: *** [deploy] Error 2
```

**Comment by achave11** [comment]
This issues can be observed when a user attempts to run `make deploy` in a fresh deployment, a user then runs `ctrl-C` immediately after any of the AWS Lambdas that have a mapping to `SQS` have generated the mapping. This Lambdas are `azul-indexer-{stagename}-index` and `azul-indexer-{stagename}-write`.

**Comment by theathorn** [comment]
Will be solved by #1546
